schemas/aws/sharing-option-1.yml

```diff
@@ -0,0 +1,43 @@
+---
+"$schema": /metaschema-1.json
+version: '1.0'
+type: object
+description: define sharing options between aws accounts
+
+additionalProperties: false
+properties:
+  "$schema":
+    type: string
+    enum:
+    - /aws/sharing-options-1.yml
+  provider:
+    type: string
+    description: type of sharing to implement
+    enum:
+    - ami
+  account:
+    "$ref": "/common-1.json#/definitions/crossref"
+    "$schemaRef": "/aws/account-1.yml"
+  regex:
+    type: string
+    description: regex expression to filter items by
```

Comment:
... by what? :D

Reply:
The items which we are in the context of?

Reply:
i just realized it is an optional field. my concern was that regex would be not flexible enough to be the way of filtering for all sharing options to come (e.g. filtering by multiple tags)